aviary/subsystems/propulsion/propeller_performance.py

```diff
@@ -214,22 +346,67 @@ def setup(self):
                 "density_ratio",
             ])
 
-        self.add_subsystem(
-            name='hamilton_standard',
-            subsys=HamiltonStandard(num_nodes=nn, aviary_options=aviary_options),
-            promotes_inputs=[
-                Dynamic.Mission.MACH,
-                "power_coefficient",
-                "advance_ratio",
-                "tip_mach",
-                Aircraft.Engine.PROPELLER_ACTIVITY_FACTOR,
-                Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICIENT,
-            ],
-            promotes_outputs=[
-                "thrust_coefficient",
-                "blade_angle",
-                "comp_tip_loss_factor",
-            ])
+        if use_propeller_map:
+            prop_model = PropellerMap('prop', aviary_options)
+            prop_file_path = aviary_options.get_val(
+                Aircraft.Engine.PROPELLER_DATA_FILE)
+            print(f"prop_file_path: {prop_file_path}")
```

Comment:
```suggestion
            if aviary_options.get_val(Settings.VERBOSITY) >= 2:
                print(f"prop_file_path: {prop_file_path}")
```
You'll have to import Settings too

Reply:
I decide to remove the print statement since it is just a debug message.